src/components/FilterTime/FilterTime.jsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { useState } from "react";
+import PropTypes from "prop-types";
+function FilterTime({ time }) {
+  let count = 0;
+  let notClicked =
+    "shadow-lg rounded-lg bg-gray-100 flex flex-col text-sm h-8 w-12 justify-center items-center border-solid border border-yellow-600 border-opacity-40 text-yellow-600";
+  let clicked =
+    "shadow-lg rounded-lg bg-yellow-600 flex flex-col text-sm h-8 w-12 justify-center items-center border-solid border border-yellow-700 text-white";
+  const [divClass, setClass] = useState(notClicked);
```

Comment:
you can do a better naming for this like 
```suggestion
  const [isClickedStyle, setIsClickedStyle] = useState(notClicked);
```